src/libraries/Common/src/System/Security/Cryptography/AesAEAD.cs

```diff
@@ -1,15 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Internal.Cryptography;
-
 namespace System.Security.Cryptography
 {
-    internal static partial class AesAEAD
+    internal static class AesAEAD
     {
-        public static void CheckKeySize(int keySizeInBytes)
+        internal static void CheckKeySize(int keySizeInBytes)
         {
-            if (keySizeInBytes != (128 / 8) && keySizeInBytes != (192 / 8) && keySizeInBytes != (256 / 8))
+            if (keySizeInBytes is not 128 / 8 and not 192 / 8 and not 256 / 8)
```

Comment:
Once you're changing the style here, shouldn't it be

```suggestion
            if (keySizeInBytes is not (128 / 8 or 192 / 8 or 256 / 8)
```
?